### ISSUE TITLE
Full-context-menu for finished downloads

### ISSUE THREAD

**Issue by JoachimHenze** [Alex313031/thorium#673]
- Thorium-legacy: Thorium_SSE3_122.0.6261.168_WINXP_x32 (I was requested to file the new feature-request against regular Thorium repository)
- on WinXPSP3 x86 (but the OS does not matter, affects all Windows versions)
- I do use the flag #disable-download-bubble so that the download-shelf is used here, but actually even that does not matter. Same problem would happen also in the "download-bubble"

The context menu for a downloaded file is inferioir.

**I would expect it to show a full context menu like I would see it within the Windows explorer.**
E.g. It is extremely annoying if you want to run "Open with" just a single time with a different app than the default.
E.g. It is extremely annoying if you want to run "File Properties".
To achieve that, you currently have to click "Show in folder, then the explorer opens, and only there you can use the proper context menu"

I am aware that a full context menu is most likely not possible as long as the download is not completed yet.
But as soon as it is completed, there is nothing speaking against a full context menu.

I would be fine with a chrome-flag or a cmd-line argument to enable that power.

Current context menu (sucks):
![image](https://github.com/Alex313031/thorium-legacy/assets/33393466/48508a07-f417-4cd3-b1c9-afa0b0063787)

This does affect all Chrome-based browsers, the problem is not limited to Thorium.
But there are browsers out there, that did that much better, e.g. ancient "Opera 12.18" browser.

**Comment by Alex313031** [comment]
@JoachimHenze I don't think it would be possible to make it open the properties window, due to restrictions on how and what Chromium can control/interact with as far as the file system and hooking into explorer.exe/shell APIs, however, the "open with" thing can probably be addressed.

And I mean, anything is possible with enough coding, but I wouldn't know where to begin with the "open properties" thing, and in any case, would subvert some of Chromium security protections (which also are applicable to Linux).
